### ISSUE TITLE
Moisture % ?

### ISSUE THREAD

**Issue by mrfunk** [fruxefarms/FruxePi#56]
Would be good to get a % reading from the analog out of the mositure sensor...  This would be so useful to get an approx idea of the dampness of the soil.

Thanks
#enhancement

**Comment by dcaud** [comment]
Agreed, do you know if the suggested hardware moisture meter permits an analog continuous reading? Do you have an example?

**Comment by mrfunk** [comment]
Yes it does have an analog output to use..

**Comment by fruxefarms** [comment]
I will investigate this, however I'm not sure I've ever been able to get any reading other than 1/0 in past development, but let me double check.

**Comment by mrfunk** [comment]
Thanks..I see you fixed the password issue, appreciate that.. how do I
update to get the fix?

**Comment by fruxefarms** [comment]
Running `git pull` while in the `FruxePi/` directory should do the trick.

**Comment by mrfunk** [comment]
Sorry, so do I just type sudo git pull? Also Fluxepi, or the master
directory?

**Comment by fruxefarms** [comment]
Yes, in your case while in the `FruxePi-master/` folder run:
```
sudo git pull 
```
If you don't have Git installed you can do so easily: 
```
sudo apt-get install git
```
Afterwards I usually reboot the containers for good measure:
```
docker restart frxpi-APACHE frxpi-PHPMYADMIN frxpi-MYSQL
```

**Comment by mrfunk** [comment]
ok done exactly what you said, unfortunately still get a 404 when trying to delete a user from this page




also when trying to edit a user i get this.


I have tried what you suggested, and it downloaded 3 files.l i have tried in both parent directories.

here is the grab,  of my sect attempt showing all upto date.. 


not sure what I’m missing, but being able to change this access is essential for me at the moment…   sorry, and thanks

DAVID

**Comment by fruxefarms** [comment]
Sorry, the delete function was absent from the last commit. That is all fixed now in that latest commit.

**Comment by mrfunk** [comment]
Nope still no joy.. done a git update again, got new files.. don’t know what I’m doing wrong. Sorry to be a pain

**Comment by fruxefarms** [comment]
What specifically is not working? Are you still not able to delete an admin? 404?

**Comment by mrfunk** [comment]
Yes can't delete it. Only make it inactive.. which I suppose is ok

**Comment by fruxefarms** [comment]
Hmm, upon further investigation I would suggest a fresh install. It is testing properly. Deleting an admin is possible. Just to clarify: To delete the hello@fruxe.co admin you must be logged in as another admin. Is that your issue perhaps?

**Comment by mrfunk** [comment]
Ok will try fresh install.. yes did logon as another user.

Thanks for your patience.


Also last question, is there a timelapse option in the works, as my camera
led randomly lights, just wondering if it's taking pictures.

Thanks

**Comment by fruxefarms** [comment]
In this latest sprint we are working on a "growlapse" function. However, at the moment the camera is taking a photo every minute and updating the background image `app/assets/img/crop_bg.jpg`.

**Comment by behinime** [comment]
I didn't get how the moisture sensor is working. 
Using the digital output I can see a "green check mark" when the soil is "watered" and red when not.
I didn't get what "Conditions"% is related to because if it's a moisture sensor % reading it gives me increasing values from 10% to 60% while testing the sensor in a glass full of water.
It's possible to use the analog output too?

**Comment by curt2199** [comment]
This tricked me up a bit as I can see my moisture sensor working but the Conditions % wasn't changing which I assumed sourced the moisture sensor however now that I see the Soil "check" and think about it being a digital output that makes sense. So what does the Conditions % do?

**Comment by LeoZanella** [comment]
The sensor just outputs a boolean value for Fruxe. But there is some analog sensors that are more durable, look for capacitive moisture sensor.
It would b a real improvement to be able to see a chart of this sensor(to watch water and nutrients consumption) and to be able to activate the pump based o the sensor value
